app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,21 @@
+class CommentsController < ApplicationController
+  def new
+    @comment = Comment.new
+  end
+
+  def create
+    @comment = Comment.new(comment_params)
```

Comment:
- Please only allow logged in users to create comments
```ruby
@comment = @current_user.comments.new(params)
```

Reply:
@afanimartin Thanks for the hints! have a nice weekend!